### PR TITLE
Forms preparations

### PR DESCRIPTION
**Pull request by datazuul**
Feature-toggle in WebsitesController in admin webapp (useReact=true). Set to false to test jQuery Website create/edit with preview image modal filedialog

### CODE REVIEW

File: dc-cudami-admin/pom.xml [hunk 1]
```diff
@@ -196,12 +196,24 @@
       <version>1.8.6</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>org.webjars.npm</groupId>
```
Comment (datazuul) [inline dc-cudami-admin/pom.xml:200]: really needed?

File: dc-cudami-admin/pom.xml [hunk 2]
```diff
@@ -220,6 +232,12 @@
       <version>4.2.2</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>org.webjars.npm</groupId>
+      <artifactId>github-com-Templarian-MaterialDesign-Webfont</artifactId>
```
Comment (datazuul) [inline dc-cudami-admin/pom.xml:237]: needed?

File: dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/security/UsersAPIController.java [hunk 3]
```diff
@@ -54,6 +58,29 @@ public BTResponse<User> find(
     return new BTResponse<>(pageResponse);
   }
 
+  @GetMapping("/api/v1/users")
```
Comment (datazuul) [inline dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/security/UsersAPIController.java:61]: only VUE

Comment (datazuul) [reply dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/security/UsersAPIController.java:61]: delete

File: dc-cudami-admin/src/main/resources/static/js/tiptap/extension-custom-hardbreak.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import { Node, mergeAttributes } from '@tiptap/core';
+
+// same as HardBreak extension: just name and class name changed to match JSON "list_item" instead "listItem"
```
Comment (datazuul) [inline dc-cudami-admin/src/main/resources/static/js/tiptap/extension-custom-hardbreak.js:3]: fix comment

File: dc-cudami-admin/src/main/resources/static/js/tiptap/tiptap-customizations.js [hunk 5]
```diff
@@ -8,12 +8,14 @@ import { Document } from '@tiptap/extension-document';
 import { Dropcursor } from '@tiptap/extension-dropcursor';
 import { Gapcursor } from '@tiptap/extension-gapcursor';
 import { HardBreak } from '@tiptap/extension-hard-break';
```
Comment (datazuul) [inline dc-cudami-admin/src/main/resources/static/js/tiptap/tiptap-customizations.js:10]: delete

File: dc-cudami-admin/src/main/resources/static/js/tiptap/tiptap-customizations.js [hunk 6]
```diff
@@ -8,12 +8,14 @@ import { Document } from '@tiptap/extension-document';
...
+// import { HardBreak } from '@tiptap/extension-hard-break';
 import { Heading } from '@tiptap/extension-heading';
 import { History } from '@tiptap/extension-history';
 import { HorizontalRule } from '@tiptap/extension-horizontal-rule';
 // import { Italic } from '@tiptap/extension-italic';
 import { Link } from '@tiptap/extension-link';
 import { ListItem } from '@tiptap/extension-list-item';
```
Comment (datazuul) [inline dc-cudami-admin/src/main/resources/static/js/tiptap/tiptap-customizations.js:17]: delete

File: dc-cudami-admin/pom.xml (hunk 1 above)
Comment (datazuul) [reply dc-cudami-admin/pom.xml:200]: done

**Review by clorenz — APPROVED** [review]

**Pull request merged**